docker-compose-prod.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.8"
+
+services:
+  app:
+    env_file: .env
+    build:
+      context: .
+      target: production
+      dockerfile: Dockerfile
+    image: todo-app/prod
+    ports:
+      - "5000:5000"
+
+# docker build --target production --tag todo_app .
+# docker run --env-file .env -p 5000:5000 todo_app
+# docker-compose up -f docker-compose_prod | stop
```

Comment:
Delete these lines, they belong in the README, not the compose file itself.

Reply:
done